nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/server/AppServerStartup.java

```diff
@@ -436,19 +422,36 @@ private boolean postStartupJob() {
             }
         }
 
+        logStartupTime();
         env.setStatus(ServerEnvironment.Status.started);
         events.send(new Event(EventTypes.SERVER_READY), false);
         pidWriter.writePidFile();
-        
+
         return true;
     }
 
+    private void logStartupTime() {
+        if (!logger.isLoggable(Level.CONFIG)) {
+            return;
+        }
+        String wallClockStart = System.getProperty("WALL_CLOCK_START");
+        if (wallClockStart != null) {
```

Comment:
Is the comment about being set only in debug no longer true?

Reply:
It was misleading, the value is set from 5 places, and not only with AS_DEBUG.